packages/admin/cms-admin/src/generator/future/generator.ts

```diff
@@ -65,7 +65,8 @@ export type GridColumnConfig<T> = (
     | { type: "dateTime" }
     | { type: "staticSelect"; values?: string[] }
     | { type: "block"; block: ImportReference }
-) & { name: UsableFields<T> } & DataGridSettings;
+) & { name: UsableFields<T>; tooltipMessage?: string } & DataGridSettings;
```

Comment:
Should this be added to the `DataGridSettings` type?

Reply:
I cannot add `tooltipMessage` to `DataGridSettings`, as it is not part of MUIs `GridColDef`.

Reply:
It seems to me, that `DataGridSettings` is exactly the type, that should contain all the settings that should be available for all columns, independent of the type-value. I don't think this has anything to do explicitly with MUIs settings.

Reply:
Got it 🫡